_posts/2015-09-30-september-newsletter.md

```diff
@@ -0,0 +1,286 @@
+projects to use it.
+
+The [first campaign][first-campaign] was very
+successful and raised about $35,000, which allowed me to work full-time on
+Neovim for roughly 6 months. Being very enthusiastic about the project and
+unable to meet all goals in those months, I continued to dedicate a very
+significant portion of my time to Neovim, so much that it started hurting my
+personal and professional life. This continued until February when I saw that I
+simply couldn't continue with my old pace. Around that time that @rappo offered
+me to test the salt platform beta version and I saw it as a way to continue my
+work on Neovim.
+
+Like it's predecessor, [the salt campaign][salt-campaign] was very successful
+and allowed me to continue Neovim contributions(in a healthy way) for the past 5
+months, thank you!
+
+### Building Neovim from source
+
+Did we ever mention how easy it is to build and install Neovim from source?
+While it has a good number of dependencies, the build system automatically
+downloads and builds everything without cluttering your system. Here's a quick
+tutorial on how to install Neovim on a unix system that has git, cmake, gnu
+autotools and a recent version of GCC:
+
+```sh
+git clone https://github.com/neovim/neovim
+cd neovim
+git checkout 0.1   # optional step
+make CMAKE_EXTRA_FLAGS="-DCMAKE_INSTALL_PREFIX=$HOME/.neovim" install
+sudo ln -s $HOME/.neovim/bin/nvim /usr/local/bin/nvim
+```
+
+And uninstalling:
+
+```sh
+rm -rf $HOME/.neovim
+sudo rm /usr/local/bin/nvim
+```
+
+That's it, go try it yourself!
+
+## Development News
+
+### ShaDa(Shared Data)
+
+@ZyX-I is working on a [major PR][shada-pr] that will completely replace the
+viminfo file for storing user data such as register contents, command history,
+variables, jump list and so on.
+
+A [known problem][shada-proposal] with viminfo is that two Vim instances running
+concurrently will override each other's data. ShaDa is a new storage format
+created by @ZyX-I that will not only this fix problem but also bring a number of
+enhancements to Neovim:
+
+- File format that
+    - Supports forward (ShaDa files from newer NeoVim can be used by older
+      versions) and backward (ShaDa files from older versions can be used by
+      newer ones without problems) compatibility, making ShaDa files
+      future-proof to a great extent;
+    - Supports hierarchical data structures, giving Neovim lot of flexibility in
+      serializing any kind of information.
+    - Is based on msgpack and explicitly standardized in documentation which
+      allows creation of plugins/tools that perform arbitrary manipulations;
+    - Assumes no state stored between data pieces inside the file, which makes
+      such tools simpler and allows such manipulations as “to concatenate two
+      ShaDa files from different NeoVim instances simply use `cat 1.shada
+      2.shada > joined.shada`, NeoVim will handle this properly when reading”;
+    - Uses absolute paths.
```

Comment:
`    - Uses expanded paths(/home/user instead of ~/).` ?

Should this be listed as an enhancement? Why is it better to use /home/user instead of ~?